maro/event_buffer/event_buffer.py

```diff
@@ -46,6 +70,17 @@ def __init__(self, disable_finished_events: bool = False):
 
         self._disable_finished_events = disable_finished_events
 
+        self._record_events = record_events
+
+        self._recorder = None
+        self._recorder_ep = None
```

Comment:
why not set the initial value to `0` ?

Reply:
to handle the first `reset()` before `env.step()`?

Reply:
yes, if default is 0, then reset first will cause first ep is 1.